hw03/src/main/java/com/otus/hw03/After.java

```diff
@@ -0,0 +1,11 @@
+package com.otus.hw03;
```

Comment:
Лучше расположить аннотации в отдельном пакете.
```suggestion
package com.otus.hw03.annotations;
```

Reply:
Готово